data/event-schemas/schema/m.room.member.yaml

```diff
@@ -136,28 +132,5 @@ properties:
     allOf:
       - $ref: "core-event-schema/unsigned_prop.yaml"
       - type: object
-        properties:
-          invite_room_state:
-            description: |-
-              A subset of the state of the room at the time of the invite, if `membership` is `invite`.
-              Note that this state is informational, and SHOULD NOT be trusted; once the client has
-              joined the room, it SHOULD fetch the live state from the server and discard the
-              invite_room_state. Also, clients must not rely on any particular state being present here;
-              they SHOULD behave properly (with possibly a degraded but not a broken experience) in
-              the absence of any particular events here. If they are set on the room, at least the
-              state for `m.room.avatar`, `m.room.canonical_alias`, `m.room.join_rules`, and `m.room.name`
-              SHOULD be included.
```

Comment:
It's a shame to lose this explanation. I suppose it could be moved elsewhere, perhaps https://spec.matrix.org/v1.4/client-server-api/#_matrixclientv3sync_invitestate?

Reply:
Actually, this advice is already included under https://spec.matrix.org/v1.4/client-server-api/#stripped-state.